guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>You can also apply a time filter using the <code>time_filter</code> method, which filters a time-enabled feature layer by datetime. When you apply the filter, the analysis will only be performed on the time filtered features. Refer to <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=time_filter#arcgis.features.FeatureLayer.time_filter" target="_blank">this page</a> for more details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>